app/views/payments/OpenPayments.scala.html

```diff
@@ -77,19 +79,28 @@ <h1 class="govuk-heading-xl">@messages("openPayments.heading")</h1>
           <p class="govuk-body">@messages("openPayments.processingTime")</p>
         </div>
 
-        <h2 class="govuk-heading-s" id="what-you-owe-missing">@messages("openPayments.whatOweHelp.heading")</h2>
-        <p class="govuk-body">@messages("openPayments.incorrect")
+        <h2 class="govuk-heading-s" id="what-you-owe-missing">
+            @if(user.isAgent) {@messages("openPayments.whatOweHelp.headingAgent")} else {@messages("openPayments.whatOweHelp.heading")}
+        </h2>
+
+        <p class="govuk-body">
+            @if(user.isAgent) {@messages("openPayments.incorrectAgent")} else {@messages("openPayments.incorrect")}
+
           <a class="govuk-link" href="https://www.gov.uk/vat-corrections" rel="noreferrer noopener" target="_blank">
-            @messages("openPayments.correctErrors")</a>.
+              @if(user.isAgent) {@messages("openPayments.correctErrorsAgent")} else {@messages("openPayments.correctErrors")}
+          </a>@messages("common.fullStop")
```

Comment:
Messages on new lines add whitespace before them, so because the full stop is on a new line to the link text before it, there is a whitespace between the two. Your unit tests show this issue too. Can you move this to be on the same line.